data/5e/wizards_of_the_coast/supplement/unearthed_arcana/class_variants/cv_classes.lst

```diff
@@ -0,0 +1,51 @@
+
+# Class Name	Hit Dice	Type			Max Level	Source Page		Define		Modify VAR																FACT
+CLASS:Bard		HD:8		TYPE:PC.BASE	MAXLEVEL:20	SOURCEPAGE:p.51	DEFINE:BardLVL|0	BONUS:VAR|BardLVL|CL	BONUS:VAR|SpellCasterTotalLVL|BardLVL	BONUS:VAR|SpellCasterClassCount|1	FACT:SpellType|Arcane
```

Comment:
This completely duplicates the bard class from the core rulebook. How is anyone going to choose between the variant and the core rulebook?